samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction Finding Periodic Advertising Locator
+#########################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The direction finding connectionless locator sample application demonstrates Bluetooth LE Direction Finding reception.
+It uses Constant Tone Extension (CTE), that is received and sampled with periodic advertising PDUs.
+
+The sample supports two Direction Finding modes:
```

Comment:
I just checked the Bluetooth core spec and see that no title case is used for direction finding, angle of arrival or angle of departure there.
I checked the existing sample (the connectionless beacon), and there we are using both title case sentence case. I cannot remember what decision regarding this (and why) we made at last review, do you?
Just so I know what we are aligning on.

Reply:
I do not remember that, let's stick to BT core spec case.